cmd/commands/subvolume.go

```diff
@@ -0,0 +1,43 @@
+package command
+
+import (
+	"github.com/rook/kubectl-rook-ceph/pkg/dr"
+	"github.com/rook/kubectl-rook-ceph/pkg/subvolume"
+	"github.com/spf13/cobra"
+)
+
+// SubvolumeCmd represents the rook commands
+var SubvolumeCmd = &cobra.Command{
+	Use:   "subvolume",
+	Short: "Calls subcommands like `ls`, `clean` and etc.",
```

Comment:
I'd suggest we keep the command descriptions generic
```suggestion
	Short: "Manage subvolumes.",
```